### DISCUSSION TITLE
Automatically adjust flex when scroll bar appears

### DISCUSSION THREAD

**Discussion by ruanformigoni** [fltk-rs/fltk-rs#1611] [Q&A] [answered]
Greetings,

Is it possible to auto adjust the flexbox (reduce the width) when the scroll bar of the scroll widget appears? The goal is to avoid the scroll bar being drawn in front of the widgets inside the scroll widget.

Thanks,

**Comment by MoAlyousef** [comment]
Hello

I think you would have to check for the addition of any other widget whether the Scroll's scrollbar is visible, and resize the flexbox to a new size.

**Reply by ruanformigoni** [reply, marked as answer]
Thanks for the reply! I decided to use `VerticalAlways`, since it was the hidden scroll bar that made things look weird.
